### PR TITLE
Flesh out introductory docs

### PR DESCRIPTION
**Pull request by thesquelched**
Add more introductory documentation.

### CODE REVIEW

File: docs/client.rst [hunk 1]
```diff
@@ -5,7 +5,8 @@ Lava Client
 
 .. currentmodule:: lavaclient
 
-The :class:`Lava` client gives you access to the CloudBigData AP
+The :class:`Lava` client provides access to all public CloudBigData API
```
Comment (PaulMontgomery) [inline docs/client.rst]: Super minor: CloudBigData -> Cloud Big Data?

File: docs/intro.rst [hunk 2]
```diff
@@ -1,9 +1,28 @@
 Introduction
 ============
 
-:mod:`lavaclient` provides python bindings for the Rackspace CloudBigData API,
-as well as a command-line client, which allows you to quickly and easily manage
-your clusters.
+:mod:`lavaclient` provides Python 2.6+ bindings for the `Rackspace CloudBigData
+API <http://www.rackspace.com/cloud/big-data>`_, as well as an easy-to-use
+command-line client.  You can use it to create clusters, manage credentials,
+interact with individual nodes, and much more.  For example, creating a Hadoop
```
Comment (PaulMontgomery) [inline docs/intro.rst]: Question for the team: Are we able to use "Hadoop" in help docs? IIRC, there are lots of restrictions on that word and I'm not sure if it applies here or not.

**Comment by PaulMontgomery** [comment]
Love the examples; just an open question and ready to upvote soon.

File: docs/client.rst (hunk 1 above)
Comment (thesquelched) [inline docs/client.rst]: I had assumed that 'CloudBigData' was the proper noun, but looking at our website, I guess it's not.

File: docs/intro.rst (hunk 2 above)
Comment (thesquelched) [inline docs/intro.rst]: The most we'd have to do is add a :tm: after each usage, but I honestly doubt that's a problem.

Comment (highlycaffeinated) [inline docs/intro.rst]: The guidelines are [here](https://www.apache.org/foundation/marks/) but the short answer is that its not a problem in this context.  We can't name our product "Cloud Hadoop" but we can be powered by Hadoop and make reference to using Hadoop clusters.

**Comment by PaulMontgomery** [comment]
+1

**Comment by rnirmal** [comment]
+1

**Pull request merged**